source/unified-test-format/unified-test-format.rst

```diff
@@ -543,11 +543,17 @@ The structure of this object is as follows:
     ``configureFailPoint`` and any commands containing sensitive information
     (per the
     `Command Monitoring <../command-monitoring/command-monitoring.rst#security>`__
-    spec).
+    spec) unless ``observeSensitiveCommands`` is true.
```

Comment:
For context, the original text of "the command(s) will be ignored in addition to ``configureFailPoint`` and any commands containing sensitive information (per the Command Monitoring spec)" was the result of my misunderstanding of that spec. Since libmongoc has historically _never_ emitted events for internal auth commands, I never realized that the _correct_ behavior was to actually emit events and just redact the command/reply documents.

Furthermore, I was oblivious to the issue described in [CDRIVER-3797](https://jira.mongodb.org/browse/CDRIVER-3797) where an event might be emitted for the createUser helper, since PHP doesn't use those helpers at all nor does it provide its own.